docs/modules/player-lifecycle/custom-player-creation.md

```diff
@@ -0,0 +1,136 @@
+<%(TOC)%>
+# Customising player creation
+
+<%(Callout message="
+Before reading this document, make sure you are familiar with:
+
+  * [Basic player spawning]({{urlRoot}}/modules/player-lifecycle/basic-player-creation)
+  * [Creating entity templates]({{urlRoot}}/reference/concepts/entity-templates)
+  * [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+")%>
+
+## Manually sending player creation requests
```

Comment:
would prefer to rephrase these titles to be "How to ..." instead of using the "-ing" form of verbs. I believe we removed that from most docs at some point. so for example

```suggestion
## How to manually send player creation requests
```